dts/bindings/sensor/maxim,max31875.yaml

```diff
@@ -0,0 +1,32 @@
+description:
+  Maxim MAX31875 Low-Power I2C Temperature Sensor.
+  Find the datasheet here https://datasheets.maximintegrated.com/en/ds/MAX31875.pdf
+
+compatible: "maxim,max31875"
+
+include: "i2c-device.yaml"
+
+properties:
+  conversions-per-second:
+    description: Number of temperature readings performed by the MAX31875 per second.
+    type: string
+    default: "4"
+    enum: # Note: the driver relies on the ordering of this enum, so change it with care.
+      - "0.25"
+      - "1"
+      - "4"
+      - "8"
+  extended-mode:
+    description: When true, use 13-bit data format to allow for measuring temperatures > 128°C
+    type: boolean
+  resolution:
+    description: |
+      The number of bits used for each temperature sample. A resolution of 10 bits takes 35ms to convert.
+      Each 1-bit increase doubles the conversion time. Similarly, each 1-bit decrease halves the conversion time.
+    type: int
+    default: 10
```

Comment:
defaults need to be justified. is sensor default? please update descriptions informing about that.

Reply:
> This is not the sensor default. I chose it arbitrarily.

That's not OK -- see the details in the link I pasted earlier for the rules on default. We only allow it in limited situations.